src/api/spec/factories/users.rb

```diff
@@ -4,6 +4,7 @@
     realname { Faker::Name.first_name }
     sequence(:login) { |n| "user_#{n}" }
     password { 'buildservice' }
+    in_rollout { false }
```

Comment:
isn't this the default?

Reply:
No, by default is `true`, because we wanted to have the new users  into the **Rollout Program**

Reply:
@hennevogel 

IMO to be deterministic the factory shouldn't rely on the default, unless the default state itself is being tested.

Reply:
Why do we turn it around in the factory then?

Reply:
We don't want to run the rollout program in all the tests. If we want to test the rollout program IMO there should be a specific test for that.
Right now, if we don't add `in_rollout: false` in the factory all the tests will be running in the `new UI`, because the enabled feature in the rollout is `Bootstrap`. Apart from that we also would need to modify the tests for `ImageTemplate`, `CloudUpload` because we are overwritten the `Feature.run_with_desactivated` with the values that are in the [feature.yml](https://github.com/openSUSE/open-build-service/pull/7868/files/f14000af0c46a7a101356b0449706ef89d012fa6#diff-c340e923ae2721e6d51a1a5da72dc438R17) file.

Reply:
@hennevogel is that ok, or do you want to see a change here?

Reply:
we've discussed this offline. @DavidKang is pushing a new version.